pkg/infra/baseAgent.go

```diff
@@ -226,6 +251,77 @@ func (a *Base) TakeFood(amountOfFood food.FoodType) (food.FoodType, error) {
 	return foodTaken, nil
 }
 
+func (a *Base) Utility() float64 {
```

Comment:
I'd prefer lower case `utility` since this is only called in the same package by tower

Reply:
sure np